floris/tools/floris_interface.py

```diff
@@ -31,10 +31,24 @@
 
 class FlorisInterface():
     """
-    The interface between a FLORIS instance and the wfc tools
-    """
+    The interface between a FLORIS instance and the Wind Farm Control (WFC)
```

Comment:
This description might be slightly out of date, maybe better something like:
FLORIS_INTERFACE provides a high-level user interface to the underlying methods